src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -57,55 +68,89 @@
 		
 		public static class PCM {}
 		
-		public static class Pneumatics {}
+		public static class Pneumatics {
```

Comment:
There was intent behind removing these constants from RobotMap. We are trying to move away from having a jumble of constants in RobotMap which cause merge conflicts and make the code unorganized. The constants should be broken up and moved into the various classes in which they are used, as `public static final ALL_CAPS` fields. 

The other intention was to reduce clutter and slowly add back what we need. As we do not have a robot yet, there is not much that should be known yet. Some of the driver configurations could be retained, but if so they should be retained in a separate class. Potentially something along the lines of `DriverConfig` or something. I believe this is implemented per the new style in the terminator branch.